app/src/main/java/com/findmyrecycling/service/ProductService.kt

```diff
@@ -10,7 +10,7 @@ import retrofit2.awaitResponse
 
 class ProductService{
 
-    suspend fun fetchProducts() : List<Product>? {
+    internal suspend fun fetchProducts() : List<Product>? {
```

Comment:
Since this service function will only be used internally, an internal modifier should be added (learned from professor's video)